src/utils/azureUtils.ts

```diff
@@ -163,6 +164,17 @@ export async function streamLogs(node: AzureRegistryTreeItem, run: AcrModels.Run
     );
 }
 
+function removeAnsiEscapeSequences(text: string): string {
+    // (^.*\x1B\[\d*;?\d*H)     - Esc[Line;ColumnH represents the cursor position. Remove any string before setting the cursore position.
```

Comment:
Is the first series of digits mandatory? In other words, is it OK if this regex matches 'Esc[H' (no digits)?

Reply:
In other words, should the first sequence of digits be matched by '\d+'?

Reply:
The documentation says below. So I the digits are optional. I will leave it as d*. Also this is the same regx used in Container Tools Window and so far it is working fine.
'Cursor Position:
Moves the cursor to the specified position (coordinates).
If you do not specify a position, the cursor moves to the home position at the upper-left corner of the screen (line 0, column 0).'

---
In reply to: [398217974](https://github.com/microsoft/vscode-docker/pull/1789#discussion_r398217974) [](ancestors = 398217974)